code/game/objects/items/blueprints.dm

```diff
@@ -19,12 +19,10 @@
 	var/const/ROOM_ERR_SPACE = -1
 	var/const/ROOM_ERR_TOOLARGE = -2
 
-/obj/item/blueprints/attack_self(mob/M as mob)
-	if (!istype(M,/mob/living/carbon/human))
```

Comment:
Noop. He was probably wrong then. User should be checked for being a proper thing.

```dm
	if (use_check(user))
		user << "This stack of blue paper means nothing to you."
		return

	add_fingerprint(user)
	interact()
```

This will do pmuch everything that's needed.